components/esp_wifi_remote/eppp/eppp_init.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "esp_log.h"
+#include "esp_wifi.h"
+#include "eppp_link.h"
+
+__attribute__((weak)) esp_netif_t *wifi_remote_eppp_init(eppp_type_t role)
```

Comment:
Can we move this init of eppp in the eppp code? \
This way the esp_wifi_remote would be cleaner and agnosttic?

Reply:
The eppp initialization doesn't belong to to the `eppp_link` component, maybe we could create a separate component from the eppp related sources? But I'd probably do it in a future versions...

Reply:
maybe, eppp_link/app or something within eppp_link?

Reply:
I'd ideally add the initialization code to the user app, but that's not according to the base requirement that the wifi code should run seamlessly on targets with or without wifi.

Reply:
Sure..